simple_mediation.rmd

```diff
@@ -0,0 +1,123 @@
+```{r, include=FALSE}
+options(tinytex.verbose = TRUE)
```

Comment:
Just a note that it's not obvious that this is just for males. Since you called the other one `_females`, maybe add `_males` to this one? Or, consider merging into one file.